### ISSUE TITLE
 An error occurred: JWT

### ISSUE THREAD

**Issue by BattouKen** [crunchy-labs/crunchy-cli#425]
**Describe the bug**
type comman crunchy-cli archive or download and:

 An error occurred: JWT error (https://cr-play-service.prd.crunchyrollsvc.com/v1/G9DUEGZ58/console/switch/play)

**To Reproduce**
ej: crunchy-cli.exe archive -a ja-JP -s es-419 https://www.crunchyroll.com/es/watch/G9DUEGZ58/from-today-i-am-a-killer
```
$ crunchy-cli.exe archive -a ja-JP -s es-419 https://www.crunchyroll.com/es/watch/G9DUEGZ58/from-today-i-am-a-killer
```

**Expected behavior**
i cant download nothing, same error with all anime.

**Screenshots**

**Client (please complete the following information):**
 - OS: [e.g. Windows 10 64bits home edition single lang]
 - Version [e.g. 3.6.4, last version crunchy.cli with scoop 

**Additional context**
![Windows PowerShell 21-05-2024 20_42_43](https://github.com/crunchy-labs/crunchy-cli/assets/81390696/cba04759-a34d-48ff-a155-c1b3cfc3be81)

**Comment by westhecool** [comment]
Multiple people were just discussing this in the Discord server. It appears that Crunchyroll has updated their app and API  consequently breaking any versions of this tool.

**Comment by BattouKen** [comment]
> Multiple people were just discussing this in the Discord server. It appears that Crunchyroll has updated their app and API consequently breaking any versions of this tool.

damm... 😢

**Comment by westhecool** [comment]
So apparently someone found a hot fix you just need to add this to the arguments: `--user-agent "Crunchyroll/1.8.0 Nintendo Switch/12.3.12.0 UE4/4.27"`

**Comment by BattouKen** [comment]
nice 

> So apparently someone found a hot fix you just need to add this to the arguments: `--user-agent "Crunchyroll/1.8.0 Nintendo Switch/12.3.12.0 UE4/4.27"`

cool 👍

**Comment by BattouKen** [comment]
para mostrar que si va perfect 👍

**Comment by AyAyEm** [comment]
It seems like a temporary fix, so i wouldn't close this yet

**Comment by westhecool** [comment]
> It seems like a temporary fix, so i wouldn't close this yet

He said that he is just going to implement this fix into the code to make it permanent.

**Comment by AyAyEm** [comment]
wouldn't it be possible that the crunchyroll app in the nintedo is outdated so they could roll an update at any time?

**Comment by patrickwuerth** [comment]
I get the following error when trying this fix: An error occurred: 
:: An error occurred: Found a stream which probably uses DRM. DRM downloads aren't supported

Does anyone else get that error message ?

**Comment by BattouKen** [comment]
> I get the following error when trying this fix: An error occurred: :: An error occurred: Found a stream which probably uses DRM. DRM downloads aren't supported
> 
> Does anyone else get that error message ?

funciona bien por ahora (mi caso) 
![Windows PowerShell 22_05_2024 11_03_39](https://github.com/crunchy-labs/crunchy-cli/assets/81390696/6e052b80-7b0a-4d6f-bdc9-31bb2e4d2de3)

trata de poner una screenshot de tu problema. Para ver si te puede ayudar alguien que sepa estos errores.

**Comment by BattouKen** [comment]
> I get the following error when trying this fix: An error occurred: :: An error occurred: Found a stream which probably uses DRM. DRM downloads aren't supported
> 
> Does anyone else get that error message ?

oh always firs try scoop update and scoop update crunchy-cli en caso de que uses scoop.

**Comment by bytedream** [comment]
The issue got fixed in [v3.6.5](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.6.5)
